usmqe/api/grafanaapi/grafanaapi.py

```diff
@@ -0,0 +1,39 @@
+
+"""
+Grafana REST API.
+"""
+
+import requests
+import pytest
+from usmqe.api.base import ApiBase
+
+LOGGER = pytest.get_logger("grafanaapi", module=True)
+
+
+class GrafanaApi(ApiBase):
+    """ Common methods for grafana REST API.
+    """
+
+    def get_dashboards(self):
+        """Get list of dashboards defined in grafana.
```

Comment:
This returns a list of dashboard names (based on the url of the dashboard), right?

Reply:
Related question: are the names returned in the list so called *slugs*? If yes, I would put a note about it here.

Reply:
Ok, added reference to documentation about `slugs`.

Reply:
I see no change in docstring of `get_dashboards` nor answer to my question (would this return list of uris, titles, slugs or what?).